fatiando/seismic/conv.py

```diff
@@ -0,0 +1,126 @@
+"""
+
+"""
+import numpy as np
```

Comment:
First thing you should import is `from __future__ import division`. This will enable the new division handling of Python 3. In Python 2 `1/2 == 0`. In Python 3 `1/2 == 0.5` and `1//2 == 0`. So `/` denotes division and `//` denotes integer division. This makes things better because you don't have to call `float` on everything.